src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -67,9 +67,8 @@
     public static final Achievement achievementBuildingColonist   = new MineColoniesAchievement("upgrade.colonist.first",
                                                                                                  0,
                                                                                                  2,
-                                                                                                 ModBlocks
-                                                                                                   .blockHutCitizen,
-                                                                                                 achievementGetSupply).registerStat();
+                                                                                                 ModBlocks.blockHutCitizen,
+                                                                                                 achievementBuildingTownhall).registerStat();
```

Comment:
Just placing it. This marks the start of a colony, even when you don't build the building.

We could add achievements for completing the townhall like we to for all buildings, and I think they would be really cool! What do you think?

Pinging @Raycoms